### ISSUE TITLE
Material Design update

### ISSUE THREAD

**Issue by Cantuariensis** [AdrienPoupa/VinylMusicPlayer#236]
Hi Adrien and the team,
I'd love to contribute to the app by updating graphics to match the new Material Design guidelines. ✨
Some of the ideas of this other Phonograph fork https://play.google.com/store/apps/details?id=code.name.monkey.retromusic are interesting but it's sometime inconsistent.

Here is my gradual proposal, easier than changing everything at the same time and less disrupting for users:
1. Update current icons to the new outline icons. I can do a selection of the icons from Google resources and design myself the missing ones. As designer I can design pixel perfect icons for different scales, and export in PNGs, SVGs and all. Then 
2. Trying some new layout in Adobe XD to discuss here and decide if it's relevant (I'm mainly thinking about the searchbox and the navigation drawer)
3. Helping developers to implement those layout changes, one by one. Sadly I'm not skilled enough to hard coding in Java.

Let me know if it works for you and I'll start right away. 😊
Thanks!

**Comment by Cantuariensis** [comment]
NB: I'd prefer to see everyone working on one or two forks instead of efforts being split in dozens of forks on the Play Store, but I guess it's hard to gather everyone on one project when the source is open? 🙄

**Comment by paolovalerdi** [comment]
You can try [Abbey](https://play.google.com/store/apps/details?id=com.paolovalerdi.abbey&hl=en), this is my fork and I'm currently working on it (And about to release source code) if you have any ideas I'm glad to hear you.

**Comment by v1r0x** [comment]
I like your proposals. And thanks for the ref to retro music player. Looks nice.

I, too, think one player instead of several forks would be great. I currently switched to retro player, because of the design. But I like that vinyl is available on f-droid. So, a nice looking foss app with f-droid releases would be awesome.

**Comment by Cantuariensis** [comment]
Thanks for your feedback. I agree we need a modern player on f-droid. Also I don't really like the idea of paying for a black theme on the majority of the other forks.
@v1r0x Would you be able to implement some changes if I make some Illustrator / Adobe XD mockups?
@AdrienPoupa Is it something you would like contributors to work on? I'm asking cause it's your fork. Otherwise we can still contribute to another fork, but this one is special. 🤗

**Comment by v1r0x** [comment]
@Cantuariensis I did some private app development a couple of years ago. Back then it was hard to follow and implement the design guidelines. But if you have some mockups and @AdrienPoupa is fine with some changes, I can look into it :)

**Comment by AdrienPoupa** [comment]
@Cantuariensis of course, I am open to contributions!

**Comment by Cantuariensis** [comment]
Finally found some time to work on some cool mockups. I will share that soon! Thanks for your patience 😁

**Comment by Octoton** [comment]
https://github.com/AdrienPoupa/VinylMusicPlayer/pull/301 is doing a small transition to a material theme (as an option so that the actual theme continue to exist)
